plugins/counter/counter.go

```diff
@@ -267,6 +267,49 @@ func (p *CounterPlugin) Message(message msg.Message) bool {
 				item.Count, item.Item))
 			return true
 		}
+	} else if len(parts) == 3 {
+		// Need to have at least 3 characters to ++ or --
+		if len(parts[0]) < 3 {
+			return false
+		}
+
+		subject := strings.ToLower(nick)
+		itemName := strings.ToLower(parts[0])[:len(parts[0])-2]
```

Comment:
Since we don't need to chop off ++ or -- here, the weird indexing math can go away.